cdap-api/src/main/java/co/cask/cdap/api/mapreduce/MapReduceTaskContext.java

```diff
@@ -35,7 +36,7 @@
  */
 @Beta
 public interface MapReduceTaskContext<KEYOUT, VALUEOUT> extends RuntimeContext, DatasetContext,
-  ServiceDiscoverer, PluginContext {
+  ServiceDiscoverer, PluginContext, LocalizationContext {
```

Comment:
Yes, it shouldn't support localizing new files, but we should be able to retrieve localized files in the task context, right?